protostar/argument_parser/argument_parser_facade.py

```diff
@@ -210,3 +255,12 @@ def __init__(self, argument_name: str, command_name: Optional[str]) -> None:
             else f"Missing required argument: `{argument_name}`"
         )
         super().__init__(self.message)
+
+
+class InconsistentInputTypesException(Exception):
```

Comment:
```suggestion
class InconsistentInputTypesException(ProtostarException):
```

Reply:
[address feedback in argument parser facade](https://github.com/software-mansion/protostar/pull/1140/commits/2dcc09754a7c9b31bb5ccb5ceedf8e8e6e625ffd)